app/views/profiles/show.html.erb

```diff
@@ -13,14 +16,24 @@
         %>
       </div>
 
-      <h1 id="profile-name" class="profile__header__name t-display">
-        <%=
-          link_to(
-            @user.display_handle, profile_path(@user.display_id),
-            class: "t-link--black"
-          )
-        %>
-      </h1>
+      <% if @user.full_name.present? %>
+        <h1 id="profile-name" class="profile__header__name t-display-full-name">
+          <%= link_to(@user.full_name, profile_path(@user.display_id), class: "t-link--black") %>
+        </h1>
+
+        <h1 id="profile-name" class="profile__header__username t-display-username">
```

Comment:
:thinking: There are 2x `h1` tags sharing the same `id`. Is that intended?

Reply:
That wasn't intended, just pushed changes to fix that, thanks :+1: